### DISCUSSION TITLE
Krokiet docker without vnc

### DISCUSSION THREAD

**Discussion by gravelfreeman** [qarmin/czkawka#1279] [Ideas]
Hello! I really like the new gui and was wondering if there could be a web version instead of a vnc version like it's the case for Czkawka. I prefer native web applications rather than vnc. Since many people self host your app, could you make it compatible for a responsive web app?

Thank you for all the great work, I'm running my first scan ever on the root of my nas, can't wait to see the results!

**Comment by fadykuzman** [comment]
Hey @gravelfreeman check this out https://github.com/qarmin/czkawka/discussions/1671
